server/models/Board.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require("mongoose");
+
+const boardSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  creation_date: {
+    type: Date,
+    default: Date.now
+  },
+
+  in_progress: {
+    type: Boolean,
+    default: true
+  },
+
+  is_completed: {
+    type: Boolean,
```

Comment:
Please use camelCase in JavaScript not snake_case. Camel case is JavaScripts convention, thanks! 😄

Reply:
I was trying to be consistent with the existing code, in the User model we had 'register_date'. I will be changing that one also.